packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -550,6 +550,7 @@ export interface SdkHttpResponse extends SdkServiceResponse {
   contentTypes?: string[];
   defaultContentType?: string;
   description?: string;
+  statusCodes: number | HttpStatusCodeRange | "*";
```

Comment:
Is there any documentation update needed for the public interface change?

Reply:
updated